Changes to "access-array-data-with-indexes.english" challenge description (#37730)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-array-data-with-indexes.english.md

## Description
<section id='description'>
We can access the data inside arrays using <dfn>indexes</dfn>.

> Array indexes are written in the same bracket notation that strings use, except that instead of specifying a character, they are specifying an entry in the array. Like strings, arrays use <dfn>zero-based</dfn> indexing, so the first element in an array is element at index <code>0</code>.

Reviewer: ```suggestion
Array indexes are written in the same bracket notation that strings use, except that instead of specifying a character, they are specifying an entry in the array. Like strings, arrays use <dfn>zero-based</dfn> indexing, so the first element in an array has an index of <code>0</code>.
```
